source/_components/media_player.blackbird.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: "Blackbird 8x8 HDMI Matrix Switch"
+description: "Instructions on how to integrate Monoprice Blackbird 4k 8x8 HDBaseT Matrix Switch into Home Assistant."
+date: 2018-03-29 16:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: monoprice.svg
+ha_category: Media Player
+ha_release: 0.67
+ha_iot_class: "Local Polling"
+---
+
+The `blackbird` platform allows you to control [Monoprice Blackbird Matrix Switch](https://www.monoprice.com/product?p_id=21819) using a serial connection.
+
+To add a Blackbird device to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: blackbird
+    type: serial
```

Comment:
Please keep the initial examples as minimal as possible.
- Remove optional parameters
- Keep (required) lists a minimal as possible.

If you'd like, you could provide an additional and more extensive example (e.g., at the end of the page).

Reply:
Thanks @frenck! I updated the example yaml as requested.